EpiAware/test/test_inference-methods.jl

```diff
@@ -0,0 +1,135 @@
+@testitem "Testing _run_manypathfinder function" begin
+    using Turing, Pathfinder
+    @model function test_model()
+        x ~ Normal(0, 1)
+        y ~ Normal(x, 1)
+    end
+
+    mdl = test_model()
+
+    # Test case 1
+    @testset "Test case 1" begin
+        nruns = 10
+        ndraws = 100
+        maxiters = 50
+
+        pfs = EpiAware._run_manypathfinder(
+            mdl; nruns = nruns, ndraws = ndraws, maxiters = maxiters)
+
+        @test length(pfs) == nruns
+        @test all(p -> p isa Union{PathfinderResult, Symbol}, pfs)
+    end
+
+    # Test case 2
+    @testset "Test case 2" begin
+        nruns = 5
+        ndraws = 50
+        maxiters = 100
+
+        pfs = EpiAware._run_manypathfinder(
+            mdl; nruns = nruns, ndraws = ndraws, maxiters = maxiters)
+
+        @test length(pfs) == nruns
+        @test all(p -> p isa Union{PathfinderResult, Symbol}, pfs)
+    end
+end
+@testitem "Testing _continue_manypathfinder! function" begin
+    using Turing, Pathfinder
+
+    @testset "Test case 1" begin
```

Comment:
Can wee say what the difference is between. these cases? they look very similar?

Reply:
I've beefed up the testing a fair bit now.